components/browser/state/src/main/java/mozilla/components/browser/state/reducer/DownloadStateReducer.kt

```diff
@@ -25,6 +25,8 @@ internal object DownloadStateReducer {
             is DownloadAction.RemoveAllDownloadsAction -> {
                 state.copy(downloads = emptyMap())
             }
+            DownloadAction.RestoreDownloadsStateAction -> state
```

Comment:
nit: no `is`

We should also add a quick test case that this action doesn't affect the state :) e.g

```
    @Test
    fun `RestoreDownloadsStateAction does nothing`() {
        val store = BrowserStore(BrowserState())

        val state = store.state
        store.dispatch(DownloadAction.RestoreDownloadsStateAction).joinBlocking()
        assertSame(store.state, state)
    }
```

Reply:
👍